lib/Server.js

```diff
@@ -302,19 +302,28 @@ function Server(compiler, options) {
 	}, this);
 
 	if(options.https) {
-		// for keep supporting CLI parameters
-		if(typeof options.https === "boolean") {
+		// pfx support
+		if(options.pfx) {
```

Comment:
I explained it a bit weird, but I meant that `options.pfx` should work the same and `options.cert`. If you'd do `--cert=mycert.crt --pfx=mypfxthingy`, the `cert` option will be ignored.